src/components/ActivityButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@mui/material';
+
+type ActivityButtonProps = {
+    color?: "inherit" | "primary" | "secondary";
+    handleClick?: () => void;
+    children?: React.ReactNode
+}
+
+const ActivityButton = (props: ActivityButtonProps): JSX.Element => {
```

Comment:
in the same way it can be written: 
`const ActivityButton = ({ color, handleClick, children}: ActivityButtonProps): JSX.Element => {`
and you can use it below by: `color={color}`

@CsabaTheKiss mentions that it may violate the DRY(Dont Repead Yourself) principle, by having a lot of references to `props`. Here it is not a problem, but if we would have a bigger file with a lot of 'props' then it could have some problems. 

Here are some references: 
https://dzone.com/articles/software-design-principles-dry-and-kisshttps://dzone.com/articles/software-design-principles-dry-and-kiss
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Good comment, feel like I need to practice this 😄 
Changed it to what you suggested